### PR TITLE
Add "Stories completed" heading

### PR DESCRIPTION
**Pull request by rahulshah7**

### CODE REVIEW

File: spec/airport_spec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require 'airport'
+
+describe Airport do
+  it { is_expected.to be_an_instance_of(Airport) }
+
+  describe '#land' do
+    it { is_expected.to respond_to(:land).with(1).argument }
+
+    context 'if plane is already landed' do
+      let(:landed_plane) { double("landed_plane", :landed => true) }
```
Comment (david-mears) [inline spec/airport_spec.rb]: great use of doubles here.

**Comment by david-mears** [comment]
There are some great tests and use of doubles here.
It seems you skipped over the first couple of user stories? Your `land` method doesn't land anything. It seems more logical to start with the basic functionality before testing for errors and edge cases.
